src/AuditConfiguration.php

```diff
@@ -19,29 +19,28 @@ class AuditConfiguration
 {
     private $auditedEntityClasses = [];
     private $globalIgnoreColumns = [];
+    private $entityIgnoredProperties = [];
     private $tablePrefix = '';
     private $tableSuffix = '_audit';
     private $revisionTableName = 'revisions';
     private $revisionFieldName = 'rev';
     private $revisionTypeFieldName = 'revtype';
     private $revisionIdFieldType = 'integer';
     private $usernameCallable;
+    private $convertEnumToString = false;
 
     /**
-     * @return AuditConfiguration
+     * @param string[] $classes
      */
-    public static function forEntities(array $classes)
+    public static function forEntities(array $classes): self
```

Comment:
We can not make changes like this for method signatures in minor versions, since they break BC.

Reply:
I added hints only for now